cmd/dcrdata/views/extras.tmpl

```diff
@@ -98,7 +98,7 @@
 				<span class="patty"></span>
 				<span class="patty"></span>
 				<span class="patty short"></span>
-				<div id="menu">
+				<div id="menu" style="max-height: 90vh; overflow-x: hidden;">
```

Comment:
I have noticed the menu cuts a lot of the menu items out of view when the current view height is less than its full height. This will ensure dcrdata users can scroll the menu because we "could" add more items(maybe) but will not be limited anymore.